docs/migration_ov_2_0/snippets/ov_api/ov_common.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/core/function.hpp>
+#include <openvino/runtime/runtime.hpp>
+
+int main() {
+    //! [ov_api_2_0:create_core]
+    ov::runtime::Core core;
+    //! [ov_api_2_0:create_core]
+
+    //! [ov_api_2_0:read_model]
```

Comment:
read_model reads 'model' and returns 'function'. Why variable name is `network`? :-)

Reply:
Just in order to use unified variable names in both examples.